Reduces repair bot health.

Pickle-Coding opened the pull request:
## About The Pull Request
Reduces the health of repair bots to 20 from 100.

## Why It's Good For The Game
They are extremely tanky while also having space mobility. For reference beepsky has 25 health and medbots 20 health. Most emagged bots try and get people out of surprise and don't have much survivability when caught out in the open. It's not really fun spending so much time dealing with a bot the roboticist can keep shitting out when you already know they're emagged.

## Changelog

:cl:
balance: Repair bots have 20 health from 100.
/:cl:

Pickle-Coding committed:
Repair bots no longer have spacewalk.

Removes the spacewalk trait from repairbot by removing it in the add_trats() call in Initialize().

archbtw0 commented:
Ided pr - Skill issue against a malf repairbot (round id 258381)

The repairbots have thrusters because they need to fly in space to fix shit.
If they drift off into space there's no way for them to go back.

Put away the salt and think about how it's going to fuck over repairbots drifting off into space.

Pickle-Coding reviewed (commented):
The roboticist can make another repair bot if one drifts off to space.

Pickle-Coding reviewed (commented):
They are expendable. The player can place rods. If they run out then skill issue.

Pickle-Coding reviewed (commented):
They're more effective at destroying the station than emagged borgs are for 0.0001* the effort and cost.

Aliceee2ch commented:
this is indeed ided, had to observe as AI

Kurgis commented:
Regular repairbot AI can't handle this change, please make the spacewalk conditional on the bot on whether it has a personality installed or not.

Pickle-Coding reviewed (commented):
> Regular repairbot AI can't handle this change, please make the spacewalk conditional on the bot on whether it has a personality installed or not. 

I'll look into that.

> Sounds like a robo player connected some neurons and had an antag idea other than building a mech and shooting people. I don't think we should punish that so harshly

They can still do that, but they won't be as oppressive with space cheese. Something as easily producable as a repair bot shouldn't take this much effort to stop their grief while they can be mass produced.

Exester509 commented:
The fuck you mean repairbots are opressive in space? They're as slow as snails and have no combat capabilities, just bash it twice with an extinguisher or something.

Pickle-Coding reviewed (commented):
> The fuck you mean repairbots are opressive in space? They're as slow as snails and have no combat capabilities, just bash it twice with an extinguisher or something. 

They have 100 health (why are they so tanky, maybe I should change that instead) and space movement significantly reduces the number of crew that can deal with them.

> Why are you using review changes to make comments instead of just commenting?

VSC does that. Out of my control.

Pickle-Coding committed:
Repair bots can spacewalk again but have less health.

Reduces the health of repairbot to 20 from 100 by reducing health and maxhealth to 20. Allows them to spacewalk by adding TRAIT_SPACEWALK to the parameters of add_trats() in Initialize().

Pickle-Coding committed:
Merge branch 'master' into removes-swarmers

Admiral-Gigaton commented:
Health reduction seems much more fair than killing the AI and mobility, I'm for this change, even if its not in good faith (*cough cough* ided)

Jacquerel commented:
As a health change this seems pretty reasonable as it had significantly more HP than other bots for no particular reason (it looks flimsier than they do, if anything).

Pickle-Coding reviewed (commented):
RP wise it has better range of motion and is more fragile due to that. Gameplay wise it has space mobility and can cause breaches so it already has good defensive properties by reducing the amount of people that can engage it. Medibot has 20 health too. Beepsky has 25. Those need to be right next to people to cause harm.

Jacquerel commented:
I was going to merge this one actually, oh well
